Userland/Applications/FileManager/FileUtils.cpp

```diff
@@ -67,4 +68,91 @@ void delete_paths(const Vector<String>& paths, bool should_confirm, GUI::Window*
     }
 }
 
+void create_trash_bin()
+{
+    auto trash_bin_path = String::formatted("{}/.trash", getenv("HOME"));
```

Comment:
This will attempt to save in `/.trash` if the user's home directory is not set in the `$HOME` environment variable.

Please use `StandardPaths::home_directory()` instead:

https://github.com/SerenityOS/serenity/blob/b25eba10d2f60c54bcd3516183050b36bfa9f390/Userland/Libraries/LibCore/StandardPaths.cpp#L17-L26

Alternatively, it may be useful to add a `trash_directory` function to `LibCore::StandardPaths`.

Reply:
Silly me I never touched the `StandardPaths` file ever!
That's a good point, Thanks for the review.